src/calibration.c

```diff
@@ -107,27 +107,44 @@ void run_static_turn_right_profile(void)
 void run_front_sensors_calibration(void)
 {
 	float linear_acceleration = get_linear_acceleration();
+	float linear_deceleration = get_linear_deceleration();
+	float required_deceleration;
 	int32_t target_micrometers;
 	int32_t micrometers_to_stop;
+	uint32_t ticks_to_stop;
 
 	disable_walls_control();
+	side_sensors_calibration();
```

Comment:
why are you using side_sensors_calibration?

Reply:
Removed. :+1: